src/Conf.ml

```diff
@@ -376,6 +377,13 @@ module Formatting = struct
     C.choice ~names ~all ~doc ~section (fun conf x ->
         {conf with break_struct= Poly.(x = `Force)} )
 
+  let compact_expressions =
+    let default = false in
+    let doc = "Style for lists and arrays." in
+    let names = ["compact-expressions"] in
```

Comment:
"expressions" is probably overly general, maybe "literals" instead?

Reply:
I don't know if "literals" is suited. I was thinking of "compact-linear-data-structures", it designates lists+arrays and not much more, but it is much longer than "compact-lists-arrays" and less explicit.

Reply:
Yes, "literal" is wrong. Perhaps "break-collection-expressions"?

Also, we could imagine applying this setting to tuples as well.